### ISSUE TITLE
Navigation in split panels doesn't work

### ISSUE THREAD

**Issue by drgarcia1986** [avelino/vim-bootstrap#86]
Some visual settings disables the command ``<Control+w>+arrows``, in my case, I just commented this [line](https://github.com/avelino/vim-bootstrap/blob/master/vim_template/vimrc#L145) :
```bash
if $COLORTERM == 'gnome-terminal'
"    set term=gnome-256color
``` 
and is working well.

**Comment by lerrua** [comment]
@drgarcia1986 what OS you use?

**Comment by drgarcia1986** [comment]
Ubuntu 14.04

**Comment by lerrua** [comment]
are you using gnome or gnome-terminal?

**Comment by drgarcia1986** [comment]
gnome-terminal

**Comment by shuhikari** [comment]
@drgarcia1986 Did you tried to setup xterm in place of gnome-256color? 

Or did you solved this in another way?

**Comment by cassiobotaro** [comment]
@drgarcia1986  Fixed in 08a9fb45.
